web/themes/contrib/civictheme/civictheme.libraries.yml

```diff
@@ -14,6 +14,16 @@ css-variables:
     theme:
       dist/civictheme.variables.css: {preprocess: false}
 
+layouts:
+  css:
+    theme:
+      dist/civictheme.layout.css: {}
+
+admin:
+  css:
+    theme:
+      dist/civictheme.admin.css: {}
```

Comment:
What's the mechanism for adding this to an admin theme?

Reply:
>this admin-style elements on a CivicTheme themed page?
yes this - I have described that